modules/rn_3_10_0.adoc

```diff
@@ -15,6 +15,11 @@ For more information, see the link:https://access.redhat.com/support/policy/upda
 [id="new-features-and-enhancements-310"]
 == {productname} new features and enhancements
 
+[id="ibm-power-z-linuxone-support"]
+===== IBM Power, IBM Z, IBM® LinuxONE support:
```

Comment:
Remove one =

```suggestion
==== IBM Power, IBM Z, IBM® LinuxONE support:
```

Reply:
Thank you @stevsmit 
I have incorporated the suggestions into the PR.